pkg/graphqlutil/generate/schema.graphql

```diff
@@ -0,0 +1,218 @@
+"Data that can be used as proof to validate notices and execute vouchers on the base layer blockchain"
```

Comment:
Move this file to a new `api` dir in the root directory because this file will be used in other places later. (Documentation, code generation for the new server implementation in Go).